samples/bluetooth/peripheral_power_profiling/boards/nrf54lv10dk_nrf54lv10a_cpuapp_ns.conf

```diff
@@ -14,3 +14,6 @@ CONFIG_NFC_NDEF_CH_MSG=n
 
 CONFIG_PM_DEVICE=y
 CONFIG_PM_DEVICE_RUNTIME=y
+
+# Workaround: RRAM always on, not needed for basic bluetooth features.
```

Comment:
Is it a workaround, or a deliberate configuration? I think we might want to remove the term "workaround," but mention that, in order to optimize power consumption, RRAM should not be forced on for basic Bluetooth LE features.